src_assets/common/assets/web/config.html

```diff
@@ -1142,6 +1242,12 @@ <h2 class="accordion-header">
               "install_steam_audio_drivers": "enabled",
               "adapter_name": "",
               "output_name": "",
+              "display_device_prep": "no_operation",
+              "resolution_change": "no_operation",
+              "manual_resolution": "",
+              "refresh_rate_change": "no_operation",
+              "manual_refresh_rate": "",
+              "hdr_prep": "no_operation",
```

Comment:
We should default `hdr_prep` to `automatic`, since an HDR host with an SDR stream is completely unusable.